apps/desktop/src/components/ProjectSettingsMenuAction.svelte

```diff
@@ -30,6 +30,11 @@
 		openExternalUrl(path);
 	});
 
+	shortcutService.on('open-in-finder', () => {
+        const path = `file://${project.path}`;
+		openExternalUrl(path);
+	});
+
 	shortcutService.on('history', () => {
```

Comment:
It appears like there is a formatting error here.

You can check for issues with `pnpm lint`, `pnpm fix` to run eslint fixes, and `pnpm format` to run prettier.